server/fixtures/models/article.js

```diff
@@ -0,0 +1,10 @@
+export const newArticle = {
```

Comment:
Remove the models folders it should just be `server/fixtures/article.js or articleData.js `